tutorial/index.md

```diff
@@ -94,9 +95,9 @@ juju grant-secret password-secret kratos
 juju run kratos/0 reset-password identity-id=<identity-id> password-secret-id=<secret:id>
 ```
 
-### Expose the identity bundle to your host machine. (only necessary if running in Multipass)
-The reason you need to expose the identity bundle to your host machine is that at the end of this tutorial
-you will need to login via a web browser.
+### Expose the identity bundle to your host machine
+
+At the end of this tutorial you'll need to log in via a web browser. Given that we're in a Multipass VM, for that to work we need to expose the identity bundle to our host machine.
```

Comment:
Can you keep something either in the head or at the beginning of the section to indicate that this is only required when running in Multipass.
For example, I don't run our tutorials in Multipass because it is very slow.

Reply:
The reason I rewrote this to presuppose Multipass is because we _tell_ people to use Multipass, so this feels more consistent. What I'm hearing is that you think I should tone down this assumption in the setup step.

Reply:
The whole setup section is about setting up an isolated environment, so I kept the Multipass VM version as the main version, but in the expose section added a note that people can skip if they're not using Multipass.